src/python/dxpy/utils/describe.py

```diff
@@ -446,7 +447,9 @@ def print_project_desc(desc, verbose=False):
     if 'containsPHI' in desc:
         print_json_field('Contains PHI', desc['containsPHI'])
     if 'databaseUIViewOnly' in desc and desc['databaseUIViewOnly']:
-        print_json_field('Database UI View Only', desc['databaseUIViewOnly'])
+        print_json_field('External Upload Restricted', desc['databaseUIViewOnly'])
```

Comment:
What are we removing "Database UI View Only" here? Looks like a copy paste error.

Reply:
Yes, sorry. I missclicked and updated different line.